spec/models/corporate_information_page_spec.rb

```diff
@@ -107,4 +112,12 @@
       expect(presented_groups.first[:id]).to eq("access-our-information")
     end
   end
+
+  describe "#contents_items" do
+    it "includes the corporate information heading in the contents list if corporate informations groups are present" do
+      expect(corporate_information_page.content_store_response["details"]["corporate_information_groups"].present?).to be(true)
```

Comment:
Should this be checking for the presence of `corporate_information_page.corporate_information_groups` instead?